install/kubernetes/cilium/templates/hubble-ui/ingress.yaml

```
@@ -11,6 +11,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  ingressClassName: {{- .Values.hubble.ui.ingress.className | default "nginx" }}
```

Help me understand why the default is "nginx".

---

because it was set at nginx in the annotations sample here: 
[        # kubernetes.io/ingress.class: nginx](https://github.com/cilium/cilium/blob/48c4b8f10b1cb1ea61b4f6cb877f80a5ff6e3833/install/kubernetes/cilium/values.yaml#L891)
if you want I can just remove the default value and the set this value in the values.yaml like this : 
https://github.com/cilium/cilium/pull/18044/files#r758686661

---

I don't think we should default to anything here, given that the legacy annotation is also not enabled by default. 

Let's do this instead:
```suggestion
{{- if .Values.hubble.ui.ingress.className }}
  ingressClassName: {{- .Values.hubble.ui.ingress.className }}
{{- end }}
```

`ingressClassName` is also not supported by Kubernetes 1.17 and older, which Cilium still supports. So we need to be careful not to populate unsupported fields by default.

---

Ok I will fix that asap thanks for the review :D